testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/MainActivity.java

```diff
@@ -40,12 +40,6 @@ protected void onCreate(Bundle savedInstanceState) {
     }
   }
 
-  public void onFirstFrameRendered() {
```

Comment:
You also want to get rid of the `implements OnFirstFrameRenderedListener` up there, as well as the import

Reply:
(lines 22 and 26)